### ISSUE TITLE
Windows Cmd display error after use vim

### ISSUE THREAD

**Issue by GavinLi369** [vim/vim#2752]
After I use vim in Windows Cmd, the previous thing in Cmd will display incorrectly like this:
![default](https://user-images.githubusercontent.com/15731822/37882395-9445f4d2-30d5-11e8-913e-ec612c6ed541.PNG)
Can someone help me?

**Comment by k-takata** [comment]
Which version do you use?
I think this has been already fixed with 8.0.0905.

**Comment by GavinLi369** [comment]
@k-takata My vim version is 8.0.586. I'm sorry this is screenshot of `gvim.exe`, because my `vim.exe` version information interface will flash over. But Their two version are the same.
![default](https://user-images.githubusercontent.com/15731822/37883265-f97f7ff2-30db-11e8-9113-d62d16378b12.PNG)

**Comment by mattn** [comment]
@GavinLi369 your vim.exe seems to be older. please try recent version.

**Comment by k-takata** [comment]
Okay, then the behavior is expected.
You may want to try the newer nightly builds at here: https://github.com/vim/vim-win32-installer/releases
Closing.

**Comment by GavinLi369** [comment]
@k-takata This is my vim with 8.0.1645. It's the same as before.
![default](https://user-images.githubusercontent.com/15731822/37884928-0a15e258-30e5-11e8-93e4-cb52e19cbc40.PNG)
![default](https://user-images.githubusercontent.com/15731822/37884932-10a4a00a-30e5-11e8-999b-16bd9fb35c72.PNG)

**Comment by mattn** [comment]
please try with `vim -u NONE -N`. If you still have this issue, please paste your smallest vimrc here to reproduce.

**Comment by k-takata** [comment]
@GavinLi369 Sorry, I don't understand what the problem is.
Some characters in the first screenshot are shown as `?`, but they are shown correctly in your latest screenshot. Isn't that the problem?

**Comment by GavinLi369** [comment]
@k-takata I'm sorry I didn't explain it clearly before. The wrong place is that after using vim, the characters displayed in cmd will be indented a distance, and the distance is not the same, like this:

![default](https://user-images.githubusercontent.com/15731822/37887294-82a40408-30f3-11e8-9e9e-d3073855d7c7.PNG)

**Comment by mattn** [comment]
@GavinLi369 What is your code page using on cmd.exe?

**Comment by GavinLi369** [comment]
@mattn  This is not code, this is just the result of using the `dir` command. This original page should be like this:

![default](https://user-images.githubusercontent.com/15731822/37901960-7de4ea9a-3125-11e8-943c-168486f840a0.PNG)

**Comment by k-takata** [comment]
I think that you are using Simplified Chinese, so the code page would be cp936.
You can check it by `chcp` command.
And, is the behavior changed if you change the font or some other console settings?

**Comment by GavinLi369** [comment]
@k-takata Yes, my code page is `936`. But when I changed the font or some other console settings, nothing changed.

**Comment by mattn** [comment]
Could you please try with LASTERFONT?
